### PR TITLE
Fixed "Does no exit" bug and miscellenous edits

### LINKED ISSUE #8: Activity runs on terminal even after stopping it from Toolbar
After running ```$ sugar-activity . ```, the activity opens and on terminal updates about scores starts to pop up; but after stopping the activity from Toolbar, terminal still waits for score updates and the process running on it doesn't stop.

### LINKED ISSUE #9: Does not exit
Tested on Ubuntu 18.04 beta with Sugar 0.112.

- [ ] when the stop button is pressed, the activity window disappears but the process does not terminate, and when used from a shell the shell prompt does not return; workaround is `^Z` followed by ~kill -9 %1`

### PR DESCRIPTION
**Pull request by yell0wfl4sh**
Fixes #8 
Fixes #9 

- added GTK version to be imported
- updated Sugargame/ files, it is now same as in similar activities
- removed "does not exit" bug

@quozl please review!

### CODE REVIEW

**Comment by yell0wfl4sh** [comment]
Also, can you please add me as a maintainer in this activity so that I can release a new version. This willl also fix #2 , also to add, I got no response from previous maintainers.

**Comment by quozl** [comment]
Invited.

https://github.com/sugarlabs/XOlympics/invitations

File: sugargame/canvas.py [hunk 1]
```diff
@@ -1,51 +1,57 @@
...
+
         self._mainwindow = mainwindow
 
         self.set_can_focus(True)
-        
+
         self._socket = Gtk.Socket()
         self.add(self._socket)
+
+        self._initialized = False
```
Comment (quozl) [inline sugargame/canvas.py]: I've since removed this `self._initialized` in Sugargame, can you use the latest sources please?

**Review by quozl — CHANGES_REQUESTED** [review]
Thanks.  Please use Sugargame v1.2?

**Comment by yell0wfl4sh** [comment]
Updated!

**Pull request merged**